gateway/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+server:
+  port: 8090
+eureka:
+  client:
```

Comment:
Please prevent the gateway from registering itself to eureka for correct load balancing later

Reply:
Thank you, I have removed this connection of the gateway with the eureka server.